src/modules/LanItems.rb

```diff
@@ -2336,6 +2336,8 @@ def createS390Device(rule)
     #  @param [Array] list of hashes describing interfaces in AY profile
     #  @return [Array] list of hashes for udev rules
     def createUdevFromIfaceName(interfaces)
+      return [] if !interfaces || interfaces.empty?
```

Comment:
btw it will work if you also use `interfaces ||= []` line, but it is just matter of taste